app/api/schema/settings.py

```diff
@@ -52,6 +52,11 @@ class Meta:
     cookie_policy = fields.Str(allow_none=True)
     cookie_policy_link = fields.Str(allow_none=True)
 
+    #
+    # CAPTCHA
+    #
+    is_google_recaptcha_enabled = fields.Bool(dump_only=True)
```

Comment:
Why is it not with other properties?

Reply:
@iamareebjamal I saw the way how various payment gateways were implemented , None of their boolean was defined in `Settings Model` , So thought to go for the same ?

Reply:
We should not repeat our mistakes

Reply:
The reason for adding an enabled field is that if I want to disable the thing temporarily, or let's say have to remove the captcha or a payment system, I should not need to remove the API keys for them

Reply:
> We should not repeat our mistakes

That was cool, 

> The reason for adding an enabled field is that if I want to disable the thing temporarily, or let's say have to remove the captcha or a payment system, I should not need to remove the API keys for them

Got it, will make appropriate changes